core/controllers/admin.py

```diff
@@ -134,6 +135,11 @@ def post(self):
             elif self.payload.get('action') == 'reload_collection':
                 collection_id = self.payload.get('collection_id')
                 self._reload_collection(collection_id)
+            elif self.payload.get('action') == 'generate_dummy_exploration':
+                dummy_exp_count = self.payload.get('dummy_exp_count')
```

Comment:
We need better names for this -- they're both counts, but one has "count" in it and the other doesn't. Clear naming is important for understandability.

How about num_dummy_exps_to_generate, num_dummy_exps_to_publish (with similar names in the frontend)?